arch/x86/kernel/head_64.S

```diff
@@ -528,11 +528,11 @@ NEXT_PAGE(zero_pud)
 	FILL(zero_pmd - __START_KERNEL_map + __PAGE_KERNEL_RO, 512)
```

Comment:
I would also be nice to prefix the zero page with kasan_ as it is used only in kasan, which is a quite special execution mode. So that kernel developers are not puzzled about its purpose.